samples/peptide_mixture_grxmc_ideal.py

```diff
@@ -19,7 +18,7 @@
 
 # Command line arguments
 
-valid_modes=["standard", "unified"]
+valid_modes=["standard", "unified", "standard-test", "unified-test"]
```

Comment:
please add a sanity check that checks that the user provide a valid mode, for example see 
`samples/Beyer2024/peptide.py